### PR TITLE
updated the bulk delete lb desc

### PR DESCRIPTION
**Pull request by dereknoblej**

### CODE REVIEW

File: api-docs/cloud-load-balancers-v1/api-reference/methods/delete-bulk-delete-load-balancers-v1.0-account-loadbalancers.rst [hunk 1]
```diff
@@ -14,6 +14,10 @@ The bulk-delete load balancer function removes the specified load balancer and
 its associated configuration from the account. Any and all configuration data is
 immediately purged and is not recoverable.
 
+To bulk-delete load balancers, provide a quesry parameter list of load balancer IDs
```
Comment (lbrackspace) [inline api-docs/cloud-load-balancers-v1/api-reference/methods/delete-bulk-delete-load-balancers-v1.0-account-loadbalancers.rst:17]: Typo: provide a query*

Comment (dereknoblej) [reply api-docs/cloud-load-balancers-v1/api-reference/methods/delete-bulk-delete-load-balancers-v1.0-account-loadbalancers.rst:17]: fixing it now, thank you.

File: api-docs/cloud-load-balancers-v1/api-reference/methods/delete-bulk-delete-load-balancers-v1.0-account-loadbalancers.rst [hunk 2]
```diff
@@ -14,6 +14,10 @@ The bulk-delete load balancer function removes the specified load balancer and
 its associated configuration from the account. Any and all configuration data is
 immediately purged and is not recoverable.
 
+To bulk-delete load balancers, provide a query parameter list of load balancer IDs
```
Comment (catlook) [inline api-docs/cloud-load-balancers-v1/api-reference/methods/delete-bulk-delete-load-balancers-v1.0-account-loadbalancers.rst:17]: To bulk-delete load balancers, provide a query parameter list of load balancer IDs.

File: api-docs/cloud-load-balancers-v1/api-reference/methods/delete-bulk-delete-load-balancers-v1.0-account-loadbalancers.rst [hunk 3]
```diff
@@ -14,6 +14,10 @@ The bulk-delete load balancer function removes the specified load balancer and
...
+(for example: /v1.0/{account}/loadbalancers?id=<id>&id=<id>). The current default limit is ten IDs
```
Comment (catlook) [inline api-docs/cloud-load-balancers-v1/api-reference/methods/delete-bulk-delete-load-balancers-v1.0-account-loadbalancers.rst:18]: For example, use `/v1.0/{account}/loadbalancers?id=<id>&id=<id>`. The current default limit is ten IDs

**Comment by dereknoblej** [comment]
build me

**Comment by catlook** [comment]
build me

**Comment by rackernexus** [comment]
Build 174 was successful. Staging content [Preview!](https://developer-revs.staging.deconst.org/build-80efeae6a/docs/cloud-load-balancers/v1/). valid for 14 days

**Pull request merged**